### ISSUE TITLE
Feature Request:

### ISSUE THREAD

**Issue by rodude123** [vsvinav/spotify-adblock-linux#14]
Hey it would be nice if it would run in the background and I wouldn't need to keep the window open

**Comment by sappelhoff** [comment]
for now you could try `nohup python adspotter &` ... **haven't tested that yet, but it might work**. After running this command, you can close the terminal and it will continue running in the background, saving all its output to a `nohup.out` file. 

Closing spotify should lead to an error with `adspotter` and thus close that process automatically.

**Comment by sappelhoff** [comment]
indeed, that seems to work :-) just make yourself an alias somewhat like:

`alias adspotter='cd ~/spotify-adblock-linux ; nohup python adspotter &'`

... assuming `~/spotify-adblock-linux` is where you have the adspotter repo.

**Comment by rodude123** [comment]
Thank you, I'll try it out

**Comment by rodude123** [comment]
hey didn't work for me 
![image](https://user-images.githubusercontent.com/10896262/78823080-03ae3e80-79d4-11ea-8759-9a5cd11049c2.png) I had still had the window open. Is there anyway I can close the window and it still work

**Comment by sappelhoff** [comment]
yes, after `nohup python adspotter &` you can just close the window

**Comment by rodude123** [comment]
Well after all this I closed the window and nothing happeened but it's ok

**Comment by jaideraf** [comment]
> alias adspotter='cd ~/spotify-adblock-linux ; nohup python adspotter &'

alias adspotter='cd ~/spotify-adblock-linux ; nohup **python3** adspotter &' to me (Ubuntu 20.04)

**Comment by ochen1** [comment]
First thing that comes to my mind is to use something like:
`./adspotter > /dev/null & disown && exit 0`

Don't know if it will work.
